src/api/v1/statuses.ts

```diff
@@ -848,18 +928,39 @@ app.post(
       .where(eq(posts.id, originalPostId));
     const fedCtx = federation.createContext(c.req.raw, undefined);
     for (const post of postList) {
+      const activity = new Undo({
+        actor: new URL(owner.account.iri),
+        object: toAnnounce(post, fedCtx),
+      });
       await fedCtx.sendActivity(
         { username: owner.handle },
         "followers",
-        new Undo({
-          actor: new URL(owner.account.iri),
-          object: toAnnounce(post, fedCtx),
-        }),
+        activity,
         {
           preferSharedInbox: true,
           excludeBaseUris: [new URL(c.req.url)],
         },
       );
+      if (!owner.account.protected) {
+        const acceptedRelays = await db.query.relays.findMany({
+          where: eq(relays.state, "accepted"),
+          with: {
+            relayServerActor: true,
+          },
+        });
+        await fedCtx.sendActivity(
+          owner,
+          acceptedRelays.map((relay) => ({
+            id: new URL(relay.relayServerActor.iri),
+            inboxId: new URL(relay.relayServerActor.inboxUrl),
+          })),
+          activity,
+          {
+            preferSharedInbox: true,
```

Comment:
We don't have to turn this on as we aren't providing recipients' shared inbox URLs here.